FFmpegInterop/FFmpegInteropLogging.h

```diff
@@ -17,21 +17,30 @@
 //*****************************************************************************
 
 #pragma once
-#include "ILogProvider.h"
 
-namespace FFmpegInterop
+#include "FFmpegInteropLogging.g.h"
+
+namespace winrt::FFmpegInterop::implementation
 {
-	public ref class FFmpegInteropLogging sealed
+	// TODO: Update for trace logging
```

Comment:
Is this still TODO? We should either track with internal work item or remove

Reply:
Yes, I'd like to get TraceLogging working with this so we can get additional tracing from FFmpeg in ETLs. I'll file a task for it.